### PR TITLE
example involving sequential runner with multiple configs

### PR DESCRIPTION
**Pull request by petermartigny**
Add an example usage of the sequential runner to run a single experiment with multiple configuration sets of parameters #33

### CODE REVIEW

File: experiments/deep_learning_with_pytorch/mlp_parameter_tuning.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+@register_plugin
+class MyReporter(ReporterABC):
+    def __init__(self):
+        self.reported = False
+
+    def report(self, name: str, experiment: ExperimentConfig, report_dir: Path):
+
+        with open(report_dir / 'metrics_report.txt', 'w') as reporting:
```
Comment (kireet) [inline experiments/deep_learning_with_pytorch/mlp_parameter_tuning.py]: maybe use smart_open here?

File: experiments/deep_learning_with_pytorch/mlp_parameter_tuning.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                    reporting.write(f"{metric}: [{', '.join([str(value) for value in values])}]\n")
+
+
+if __name__ == "__main__":
+
+    dir = Path(__file__).parent
+    home_env = str(Path.home() / 'work/transfer-nlp-data')
+    ExperimentRunner.run_all(experiment=dir / 'mlp_parameter_tuning.json',
+                             experiment_config=dir / 'mlp_parameter_tuning.cfg',
+                             report_dir=dir / 'reports',
```
Comment (kireet) [inline experiments/deep_learning_with_pytorch/mlp_parameter_tuning.py]: probably not great to create directories in the repo area that aren't meant to be checked in. i would suggest using something under `home_env` instead. also, i think we should use a timestamp dir under reports to indicate when the code ran as best practice. so finally i would suggest something like
`home_env / 'mlp_parameter_fine_tuning' / 20190521` as the report dir where 20190521 is the current date.

Comment (kireet) [reply experiments/deep_learning_with_pytorch/mlp_parameter_tuning.py]: also probably don't use `dir` as a variable name: https://docs.python.org/3/library/functions.html#dir

**Comment by petermartigny** [comment]
close #33

**Pull request merged**